README.md

```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+## Who am I? 
+My name is Liam Cohen. I am currently a **rising third year** in the College
+majoring in *Latin American Studies and Political Science*. I highly recommend 
+that you check out the 
+[Center for Latin American Studies](https://clas.uchicago.edu/). 
+There are great professors associated with the Center. 
+
+## academic interests 
+- Democracy 
+- Political (In-)Stability 
```

Comment:
I recommend leaving a blank line between different Markdown elements, such as between headers and paragraph text. It avoids rendering issues in more complex documents